Added a committee effect subsection (#51)
chhoumann/p5: sections/Chapter 1 - Introduction to knox/50_Committees.tex

During Scrum-of-Scrums, it was agreed upon that each team would collaborate on parts of the written project. The meetings were held in order to collaborate on writing these parts, to correct the written material, and ensure that it lived up to collective standards of writing.

% Indsæt UI

\subsection{Committees effect on the sprint planning}
\textit{This section is not part of the collective writing.}

> One of the main goals with establishing committees that overlooked most of the main parts of the \knox{} project was to get ahead of potential future problems that could arise with multiple teams working on the same project. 

Reviewer: ```suggestion
One of the main goals of establishing committees that surveyed most of the \knox{} project was to get ahead of potential problems that could arise with multiple teams working on the same project. 
```

> This was done by escalating decisions that would have a large affect on the development teams to these committees and allowing them to take a decisions with the input from all other teams. 

Reviewer: ```suggestion
This was done by discussing decisions that would have a large effect on the development teams to these committees and allowing them to make decisions with the input from all other  \knox{} teams as well as the Product Owner of \knox{}. 
```

> The wanted outcome from this was to make the planning phase easier for the groups due to them knowing when other teams expected to develop and release components they relied on.

Reviewer: ```suggestion
The wanted outcome was to make the planning phase easier for the groups, by giving them knowledge of what other teams expected to develop and release, as well as which components the various teams depended on.
```

> A downside to this approach was that it meant the development teams could not actively take large decisions themselves, and instead often had to wait on an official decision from the committees. 

Reviewer: ```suggestion
A downside to this approach was that the development teams could not actively make impactful decisions themselves, and instead often had to wait on an official decision from the committees, slowing down development speed. 
```

> Another example was that one of the primary goals being to create a UI component. 

Reviewer: ```suggestion
For instance, one of the primary goals was to create a UI component. 
```

> This task was locked behind the UI committee taking a decisions on how to move forward with the UI. 

Reviewer: ```suggestion
This task was locked behind the UI committee making decisions on how to move forward with the UI. 
```

> Teams could not start working on their component in the old setup due to the UI committee not initially knowing if it wanted to keep the old UI or wanted to change framework and build a new one up from scratch.

Reviewer: ```suggestion
Teams could not start working on their component in the old setup due to the UI committee not initially knowing if it wanted to keep the old UI or wanted to change the framework and build a new one up from scratch.
```
